src/ui/components/TestSuite/suspense/TestItemsCache.ts

```diff
@@ -128,6 +130,12 @@ export const TestItemsCache = createSingleEntryCache<
         scopePath,
       };
     });
+
+    pingTelemetry("TestItemsCacheLoad", {
```

Comment:
Ideally - this would be baked into caches so that you can just toggle telemetry on or off...

Reply:
Okay, let's do that then. I'll post a follow up PR.

Reply:
Maybe something like #9261?